build_scripts/es_setup/elasticsearch.yml

```diff
@@ -52,7 +52,7 @@ path.logs: /data/es_logs
 #
 # Set the bind address to a specific IP (IPv4 or IPv6):
 #
-network.host: 0.0.0.0
+network.host: 127.0.0.1
```

Comment:
This would only work for single server install, right? Otherwise, how can we access ElasticSearch?

Reply:
Changed it back to 0.0.0.0